### DISCUSSION TITLE
docker container open port 11434 for external access of other apps

### DISCUSSION THREAD

**Discussion by andreascschmidt** [open-webui/open-webui#7494] [Q&A]
May I ask please if anyone can hint me please how I can make the port 11434 from ollama server in the docker image external available for other applications like perplexica. 

In the docker-compose I have already -p 11434:11434 to present it but inside I noted in netstat 

tcp        0      0 127.0.0.1:11434         0.0.0.0:*               LISTEN      13/ollama           

which makes me think is the reason why I can't connect to ollama on that port from another application

**Comment by Qanop** [comment]
I was looking half day for this info, but I found it. This setting is dependent on `OLLAMA_HOST` environment, that is required on `ollama` image. 
> services:
  open-webui:
    image: ghcr.io/open-webui/open-webui:ollama
    restart: unless-stopped
    container_name: open-webui
    environment:
      - OLLAMA_HOST=0.0.0.0
    volumes:
      - ./open-webui:/app/backend/data
      - ./ollama:/root/.ollama
    ports:
      - 9472:8080
      - ${OLLAMA_WEBAPI_PORT-11434}:11434

If you working on `ghcr.io/open-webui/open-webui:ollama` image, pass this environment directly into this 2in1 image, but if you have open-webui and ollama separated, pass it to `ollama` image.

**Reply by RhadenGG** [reply]
You are a G! Thank you
